docs/pages/deep-sleep-manager_L3_TestProcedure.md

```diff
@@ -0,0 +1,209 @@
+          - "chmod +x /opt/HAL/deepsleep_L3/hal_test"
+          - "chmod +x /opt/HAL/deepsleep_L3/run.sh"
+          - cp -rf /usr/lib/libiarmmgrs-deepsleep-hal.so /opt/HAL/deepsleep_L3/
+          - "ln -s /usr/lib/libiarmmgrs-deepsleep-hal.so /opt/HAL/deepsleep_L3/libiarmmgrs-deepsleep-hal.so"
+          - cp -rf /usr/lib/libiarmmgrs-power-hal.so /opt/HAL/power_L3/
+          - "ln -s /usr/lib/libiarmmgrs-power-hal.so /opt/HAL/power_L3/libiarmmgrs-power-hal.so"
+        streams:
+        test1_TriggerDeepsleep:
+          <<: *defaults
+          streams:
+```
+#### Test Suite Configuration
+Example Test Setup configuration File: [deepsleep_test_suite.yml](../../../host/tests/deepsleepClasses/deepsleep_test_suite.yml)
+Example Test Setup configuration File: [power_test_suite.yml](../../../host/tests/deepsleepClasses/power_test_suite.yml)
+Update the execute command according to the device path where `HAL` binaries are copied.
+
+```yaml
+deepsleep:
+  description: "Deepsleep manager testing profile"
+  test:
+    execute: "/tmp/run.sh -p /tmp/deepsleepmanagerExtendedEnumsNotSupported.yaml"
+    type: UT-C  # Cunit tests (UT-C)
+```
+
+```yaml
+power:
+  description: "Power manager testing profile"
+  test:
+    execute: "/tmp/run.sh -p /tmp/deepsleepmanagerExtendedEnumsNotSupported.yaml"
+    type: UT-C  # Cunit tests (UT-C)
+```
+
+## Run Test Cases
+
+Once the environment is set up, you can execute the test cases with the following command
+
+```bash
+python <TestCaseName.py> --config </PATH>/ut/host/tests/configs/example_rack_config.yml --deviceConfig </PATH>/ut/host/tests/configs/deviceConfig.yml
+```
+
+## Test Setup Connections
+
+Verify that the enable deepsleep sources are working on the device
+For Example:
+
+- Set the deepsleep wake up source to on via power manager api, and then trigger deepsleep, and subsequent wakeup
+
+## Test Cases
+### test1_TriggerDeepsleep.py
+
+#### Platform Support - test01
+
+- Sink/Source
+
+#### User Input Required - test01
+
+**Yes**: User interaction is necessary to trigger deepsleep and trigger wake up (This will be automated later).
+
+#### Acceptance Criteria - test01
+
+No tests are skipped, and all wake up sources are being triggered properly
+
+#### Expected Results - test01
+
+All wake up sources should wake up properly
+
+#### Test Steps - test01
+
+- Initiate the Test:
+
+  - Select and execute the Python file: **`test1_TriggerDeepsleep.py`**
+  - The test will automatically download all required artifacts, copying them to the designated target directory before commencing execution.
+
+- Trigger deepsleep prompt:
+
+    Would you like to trigger WAKE-UP SOURCE test? Note, skipping the test is considered a fail condition. (Y/N):
+
+    Please trigger wake up via [will be mention whatever wake up source is being tested]. Did the device wakeup as expected (Y/N):
+
+- The above will loop through all wake up sources that acclicable for the specific device based on the .yaml file.
```

Comment:
Add details of manual triggering of wakeup. How to wakeup device and how to verify

Reply:
Updated, with how to trigger, and how to verify.